config/routes.rb

```diff
@@ -184,14 +181,14 @@
   end
 
   # Facebook
-  get '/channel.html' => 'facebook#channel', :as => :facebook_channel
+  get '/channel.html' => 'facebook#channel', as: :facebook_channel
 
   # Exceptions
-  get "/404", :to => "errors#not_found"
-  get "/500", :to => "errors#error"
-  get "/errors/crash", :to => "errors#test_exception"
-  get "/errors/maintenance", :to => "errors#maintenance"
-  get "/errors/heroku", :to => "errors#test_heroku"
+  get "/404" => "errors#not_found"
+  get "/500" => "errors#error"
+  get "/errors/crash" => "errors#test_exception"
+  get "/errors/maintenance" => "errors#maintenance"
```

Comment:
Line is too long. [86/80]<br>Unnecessary spacing detected.<br>Put one space between the method name and the first argument.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.